configs/modules/t31l_sc2336_atbm6031_defconfig

```diff
@@ -3,3 +3,6 @@ BR2_SOC_INGENIC_T31L=y
 BR2_SENSOR_MODEL="sc2336"
 BR2_PACKAGE_WIFI=y
 BR2_PACKAGE_WIFI_ATBM6031=y
+CONFIG_KERNEL_VERSION_4=y
+KERNEL_VERSION_4=y
+BR2_KERNEL_VERSION_4=y
```

Comment:
Do we need all three? `KERNEL_VERSION_4=y` is enough to define the kernel version for makefile and packages.

Also, you need to create a separate profile for kernel 4 version in `config/testing/`, this one should stay for 3.10 as legacy/stable.